package.json

```diff
@@ -43,11 +43,14 @@
   "private": true,
   "homepage": "https://github.com/nextcloud/news",
   "dependencies": {
+    "@nextcloud/auth": "^2.0.0",
     "@nextcloud/axios": "^1.11.0",
     "@nextcloud/dialogs": "^3.1.2",
     "@nextcloud/initial-state": "^2.0.0",
+    "@nextcloud/l10n": "^1.6.0",
     "@nextcloud/router": "^2.0.0",
     "@nextcloud/vue": "^7.0.1",
+    "regenerator-runtime": "^0.13.11",
```

Comment:
Wouldn't `regenerator-runtime` rather be a dev-dependency as it's apparently only used in the unit tests?

Reply:
that's a good point, I was just trying to fix the error and didn't think about how it was included in the package.json